packages/p2p/src/stores/buy-sell-store.js

```diff
@@ -694,35 +697,46 @@ export default class BuySellStore extends BaseStore {
         return errors;
     }
 
+    handleAdvertInfoResponse(response) {
+        if(response.error) return;
```

Comment:
shouldn't we display the error message here for the users to see what error they got?

Reply:
i'm not sure what was the requirement. the below code was there. 
`// Added a check to prevent console errors
                                if (response?.error) return;`

i moved it to a separate function. we might require design for changing it.

Reply:
ah alright, i think for now we can have it as is, just add a TODO to see if we need to display it and then try to ask the squad if we should be displaying it 👍

Reply:
@nada-deriv why did we remove checking of response?

Reply:
@farrah-deriv .. no .. we haven't removed it.. i moved the code to a separate functino.. but the error handling is not done for the response.. we would need a design for that. 
```
    handleAdvertInfoResponse(response) {
        //TODO: error handling for response
        if(response.error) return;
        const { p2p_advert_info } = response;
        if (this.selected_ad_state?.id === p2p_advert_info.id) {
            this.setSelectedAdState(p2p_advert_info);
        }
    }
```